Packages/com.unity.inputsystem/InputSystem/Utilities/TypeTable.cs

```diff
@@ -65,5 +70,20 @@ public Type LookupTypeRegistration(string name)
                 return type;
             return null;
         }
+
+        #if UNITY_EDITOR
+        public bool ShouldHideInUI(string name)
```

Comment:
This isn't used ATM. Was from an aborted attempt that added a new type of processor for internal use. Left the code in here because I think the functionality is actually generally useful (for layouts, we already have and use something similar).